meson.build

```diff
@@ -0,0 +1,38 @@
+project('shorah',
+    ['c', 'cpp'],
+    version : '1.1.1',
+    default_options : [
+        'buildtype=release',
+        'default_library=shared',
+        'warning_level=3'],
+    license : 'GPL')
+
+# info
+as_version = meson.project_version()
+ver_arr = as_version.split('.')
+as_major_version = ver_arr[0]
+as_minor_version = ver_arr[1]
+as_micro_version = ver_arr[2]
+
+message('---------- Configure python scripts ----------')
+root_dir = meson.source_root()
+conf_data = configuration_data()
+conf_data.set('PYTHON', 'python3')
```

Comment:
this means you wont be able to change the python interpreter. Meson currently already has a python module: http://mesonbuild.com/Python-3-module.html together with `find_python` and `sysconfig_path` you can avoid the manual `shorah-install.sh` script